### DISCUSSION TITLE
Save custom dynamic config changes permanently

### DISCUSSION THREAD

**Discussion by alex98721** [darkreader/darkreader#11346] [Q&A]
I added a line in the Dark Reader dev tools window that I want to stay there even after I close the current Firefox session. But the dynamic config file gets reset each time, so I wonder how to access the local version of it that is presumably stored somewhere on my PC and add lines of CSS code for the dark reader extension to follow for as long as the edits have been made. Is there also a way to prevent those edits from being overridden when the extension updates?

**Comment by dgstangel** [comment]
+100!  I just started using DarkReader, and I was delighted to find the Dev Tools feature because parts of one of my enterprise web applications (Mitratech TeamConnect, if anyone cares) does not respond effectively to the native inverting logic;  so I added a bunch of custom CSS to the Dev Tools interface, only to find to my frustration that the changes were lost as soon as I restarted!

**Comment by nvd916** [comment]
Would love to have this also. What works for me in a very specific situation might not work for the majority of people.
